products/ros.md

```diff
@@ -18,20 +18,28 @@ releases:
     releaseDate: 2020-05-23
     eol: 2025-05-01
 
+    latestReleaseDate: 2020-05-23
```

Comment:
Looks like it was not necessary to remove it in #2925. But I still think the column should not be displayed.